lib/doctor.js

```diff
@@ -98,12 +98,12 @@ function makePretty (p) {
   if (ping !== 'ok') list[0][2] = 'Check your internet connection'
   if (!semver.satisfies(npmV, '>=' + npmLTS)) list[1][2] = 'Use npm v' + npmLTS
   if (!semver.satisfies(nodeV, '>=' + nodeLTS)) list[2][2] = 'Use node v' + nodeLTS
-  if (registry !== defaultRegistry) list[3][2] = 'Try `npm config set ' + defaultRegistry
-  if (whichGit === 'not installed') list[4][2] = 'Install git'
-  if (readbleCaches !== 'ok') list[5][2] = 'Check the permission of your files in ' + npm.config.get('cache')
-  if (executableGlobalModules !== 'ok') list[6][2] = 'Check the permission of your files in ' + globalNodeModules
-  if (executableLocalModules !== 'ok') list[7][2] = 'Check the permission of your files in ' + localNodeModules
-  if (checksumCachedFiles !== 'ok') list[8][2] = 'You have some broken packages in your cache'
+  if (registry !== defaultRegistry) list[3][2] = 'Try `npm config set registry ' + defaultRegistry
```

Comment:
Since this patch is already added into v4.1.0, I've made a patch for it. Will be fixed :)
+ https://github.com/npm/npm/pull/15402